src/lib.rs

```diff
@@ -1254,6 +1254,35 @@ pub unsafe trait TryFromBytes {
 
         candidate.map(MaybeAligned::as_ref)
     }
+
+    /// Attempts to read a `Self` from a byte slice.
+    ///
+    /// `try_read_from` validates that `bytes` contains a valid `Self`. If it
+    /// does, those bytes are read out of `bytes` and reinterpreted as a `Self`.
+    ///
+    /// Note that Rust's bit validity rules are still being decided. As such,
+    /// there exist types whose bit validity is ambiguous. See the
+    /// `TryFromBytes` docs for a discussion of how these cases are handled.
+    #[inline]
+    #[doc(hidden)] // TODO(#5): Finalize name before remove this attribute.
+    fn try_read_from(bytes: &[u8]) -> Option<Self>
+    where
+        Self: Sized,
+    {
+        let candidate = MaybeUninit::<Self>::read_from(bytes)?;
+        let cptr = Ptr::from_maybe_uninit_ref(&candidate);
```

Comment:
The nittiest of nits, but I think this is slightly more in keeping with our snake case convention:
```suggestion
        let c_ptr = Ptr::from_maybe_uninit_ref(&candidate);
```

Reply:
Done.